### PR TITLE
feat(nimbus): Include fetch summary in update_external_resources PR

### LINKED ISSUE #9815: Report fetch summary in PRs
The update external configs task now is importing a lot more manifests at once and its hard to keep track of whats going on. Additionally, sometimes individual fetches fail, which doesnt bring down the whole job (see [https://github.com/mozilla/experimenter/issues/9619](https://github.com/mozilla/experimenter/issues/9619|smart-link)).

The fetch command generates a summary, which we should include in the generated PR so we can tell what is going on in the fetch task. If a PR isnt generated (e.g., because all the fetches fail), we should instead hard fail the CI task so we get an alert and can fix it.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-4069)

### PR DESCRIPTION
**Pull request by brennie**
Because

- fetching feature manifests can fail (see #9804 for an example);
- the fetch command does not hard fail when an individual fetch fails; and
- we are generating a summary of all the fetches

This commit

- updates the fetch command to hard fail if there are fetch failures and no
  successes (i.e., in the case where it would not open or update a PR);
- adds a flag to the fetch command to write the summary to a file; and
- opens and updates PRs with a summary of the fetches in the PR description

Fixes #9815

### CODE REVIEW

**Review by jaredlockhart — COMMENTED** [review]
Oh heck yeah.  Can you think of way to force this so we can test it before we land it?  Like remove the main filter and temporarily futz the diff check to force it to create a PR just so we can see what it'll spit out before we land it?

**Comment by brennie** [comment]
See https://github.com/mozilla/experimenter/pull/9821

**Review by jaredlockhart — APPROVED** [review]
So cool!!  We should later extend to include more information about the rich diff, like fields types etc.  Thnx @brennie 🎉 🎉 🎉 🎉

**Comment by brennie** [comment]
Blocked on #9826

**Pull request merged**
